Firestore/Source/API/FIRDocumentSnapshot.m

```diff
@@ -211,6 +215,14 @@ - (id)convertedValue:(FSTFieldValue *)value options:(FSTFieldValueOptions *)opti
   return result;
 }
 
+- (nullable FIRTimestamp*)getTimestamp:(id)key {
```

Comment:
In general Objective-C naming is obtuse. Full guide here: https://developer.apple.com/library/content/documentation/Cocoa/Conceptual/CodingGuidelines/Articles/NamingMethods.html

Also, see Google-specific guidance here: http://google.github.io/styleguide/objcguide.html

Briefly, for a method that returns a value the label should start by describing that thing. Arguments get their own part of the label separated from the return value with a preposition. "get" as a prefix is almost never used.

Taken together with the feedback above that the argument is a field and not a key, this should likely be named something like `timestampForField`.

Reply:
Done.